### DISCUSSION TITLE
Hi, which tool do you use for the documentation site ?

### DISCUSSION THREAD

**Discussion by ghost** [xyflow/xyflow#2797] [General]
I'm looking for inspiration for my documentation site. It would be great if you could point to the doc template used for the site.

Thanks,
Vetrivel

**Comment by bcakmakoglu** [comment]
[Docusaurus.](https://docusaurus.io/)

**Reply by bcakmakoglu** [reply]
That's the tool that's used for the react-flow-docs :)

**Reply by ghost** [reply]
Thanks for your prompt response.

**Reply by moklick** [reply]
The website is open source too https://github.com/wbkd/react-flow-docs
